src/kibana/components/index_patterns/_field.js

```diff
@@ -38,7 +38,7 @@ define(function (require) {
 
       var indexed = !!spec.indexed;
       var scripted = !!spec.scripted;
-      var sortable = indexed && type.sortable;
+      var sortable = spec.name === '_score' || (indexed && type.sortable);
```

Comment:
Not a show-stopper, but adding an exception by field name here is unfortunate. Since this field is being forced into the mapping, it seems like there should be a property we could set that would indicate that it's sortable.